src/main/java/it/smartcommunitylab/aac/saml/provider/SamlIdentityProviderConfigMap.java

```diff
@@ -250,6 +252,15 @@ public void setRequireEmailAddress(Boolean requireEmailAddress) {
         this.requireEmailAddress = requireEmailAddress;
     }
 
+    public String getSubAttributeName() {
```

Comment:
you shoud also update SamlIdentityProviderConfig and add a method to export the subAttributeName. Keep null as default value.